samples/src/jvmTest/java/okio/samples/ChannelsTest.java

```diff
@@ -84,7 +83,7 @@
   }
 
   @Test public void testReadWriteFile() throws Exception {
-    Path path = temporaryFolder.newFile().toPath();
+    java.nio.file.Path path = temporaryFolder.newFile().toPath();
```

Comment:
explicit package on purpose? It's only a sample so whatever.
You could also alias import to `NioPath`

Reply:
Tragically no import aliases in `.java` files! I wanted to avoid using `Path` here ’cause locally it looks like our `okio.Path` type.

Reply:
totally missed the elephant in the room.